mev_inspect/schemas/classifiers.py

```diff
@@ -14,6 +16,28 @@ def get_classification() -> Classification:
         raise NotImplementedError()
 
 
+class PunkBidClassifier(Classifier):
+    @staticmethod
+    def get_classification() -> Classification:
+        return Classification.punk_bid
+
+    @staticmethod
+    @abstractmethod
+    def get_bid(trace: DecodedCallTrace) -> PunkBid:
+        raise NotImplementedError()
+
+
+class PunkBidAcceptanceClassifier(Classifier):
+    @staticmethod
+    def get_classification() -> Classification:
+        return Classification.punk_accept_bid
+
+    @staticmethod
+    @abstractmethod
+    def get_accept_bid(trace: DecodedCallTrace) -> PunkBidAcceptance:
```

Comment:
might be a little nicer to move this logic into here: https://github.com/flashbots/mev-inspect-py/pull/138/files?authenticity_token=mYvoUHaaHM4VFXvE7%2BWM7nTzfbV%2Fr%2Fw0jGpRmzeHRcqkMdfn78Ojuv52AFzQO63fltI7QGNHdkEs%2B2itE3r3cg%3D%3D&file-filters%5B%5D=.py&file-filters%5B%5D=No+extension#diff-b7d464d841fd2c44691731a28b6265cb993eeec1f621e2a7fd8526556191a905R59

Reply:
then call the classifier

little bit awkward since there's only one protocol that implements the `PunkBidAcceptance` classifier, but that'd be more consistent